example/gardener-local/kind/cluster/templates/_kubeadm_config_patches.tpl

```diff
@@ -13,18 +12,21 @@
     - gardener-apiserver.relay.svc.cluster.local
 {{- end }}
     extraArgs:
-{{- if or (not .Values.gardener.controlPlane.deployed) (not .Values.gardener.controlPlane.kindIsGardenCluster) }}
-      authorization-mode: RBAC,Node
-{{- else }}
-      authorization-mode: RBAC,Node
-      authorization-webhook-cache-authorized-ttl: "0"
-      authorization-webhook-cache-unauthorized-ttl: "0"
+      authorization-config: /etc/gardener-local/kube-apiserver/authz-config.yaml
     extraVolumes:
-    - name: gardener
-      hostPath: /etc/gardener/controlplane/auth-webhook-kubeconfig-{{ if eq .Values.networking.ipFamily "dual" }}ipv4{{ else }}{{ .Values.networking.ipFamily }}{{ end }}.yaml
-      mountPath: /etc/gardener/controlplane/auth-webhook-kubeconfig-{{ if eq .Values.networking.ipFamily "dual" }}ipv4{{ else }}{{ .Values.networking.ipFamily }}{{ end }}.yaml
+    - name: authz-config
+      mountPath: /etc/gardener-local/kube-apiserver/authz-config.yaml
+      readOnly: true
+      pathType: File
+{{- if and .Values.gardener.controlPlane.deployed .Values.gardener.controlPlane.kindIsGardenCluster }}
+      hostPath: /etc/gardener-local/kube-apiserver/authz-config-with-seedauthorizer.yaml
+    - name: authz-webhook-kubeconfig
+      mountPath: /etc/gardener-local/kube-apiserver/authz-webhook-kubeconfig.yaml
+      hostPath: /etc/gardener-local/kube-apiserver/authz-webhook-kubeconfig-{{ if eq .Values.networking.ipFamily "dual" }}ipv4{{ else }}{{ .Values.networking.ipFamily }}{{ end }}.yaml
```

Comment:
Previously either [this](https://github.com/gardener/gardener/blob/master/example/gardener-local/kind/cluster/values-dual.yaml) or [this](https://github.com/gardener/gardener/blob/master/example/gardener-local/kind/cluster/values-ipv6.yaml) file was mounted, valued from which are being used in `cluster.yaml`.

How is it working now without it, do we not need those files now?

Reply:
The mount path in the container is always the same. The only difference is which file from the host is mounted: https://github.com/gardener/gardener/pull/10984/files#diff-bec1b220a0fda1d53eb01161e41b4ff2f035201444acad1e2382fbab7738071aR25-R29

Reply:
I misread the deleted lines.